### ISSUE TITLE
app crash on startup because of Unknown color

### ISSUE THREAD

**Issue by mahibi** [nextcloud/talk-android#2633]
### Steps to reproduce

got this for a testaccount on some nextcloud server

### Expected behaviour

no crash

### Actual behaviour

app crash when logging in:

```
2022-12-16 13:11:01.524 20219-20219 AndroidRuntime          com.nextcloud.talk2                  E  FATAL EXCEPTION: main
                                                                                                    Process: com.nextcloud.talk2, PID: 20219
                                                                                                    java.lang.IllegalArgumentException: Unknown color
                                                                                                    	at android.graphics.Color.parseColor(Color.java:1391)
                                                                                                    	at com.nextcloud.android.common.ui.color.ColorUtil.parseColorOrFallback(ColorUtil.kt:76)
                                                                                                    	at com.nextcloud.android.common.ui.color.ColorUtil.getNullSafeColorWithFallbackRes(ColorUtil.kt:44)
                                                                                                    	at com.nextcloud.talk.ui.theme.ServerThemeImpl.<init>(ServerThemeImpl.kt:41)
                                                                                                    	at com.nextcloud.talk.ui.theme.MaterialSchemesProviderImpl.getMaterialSchemesForCapabilities(MaterialSchemesProviderImpl.kt:60)
                                                                                                    	at com.nextcloud.talk.ui.theme.MaterialSchemesProviderImpl.getMaterialSchemesForUser(MaterialSchemesProviderImpl.kt:49)
                                                                                                    	at com.nextcloud.talk.ui.theme.MaterialSchemesProviderImpl.getMaterialSchemesForCurrentUser(MaterialSchemesProviderImpl.kt:56)
                                                                                                    	at com.nextcloud.talk.ui.theme.ThemeModule$Companion.provideCurrentMaterialSchemes(ThemeModule.kt:42)
                                                                                                    	at com.nextcloud.talk.ui.theme.ThemeModule_Companion_ProvideCurrentMaterialSchemesFactory.provideCurrentMaterialSchemes(ThemeModule_Companion_ProvideCurrentMaterialSchemesFactory.java:43)
                                                                                                    	at com.nextcloud.talk.application.DaggerNextcloudTalkApplicationComponent$NextcloudTalkApplicationComponentImpl.materialSchemes(DaggerNextcloudTalkApplicationComponent.java:432)
                                                                                                    	at com.nextcloud.talk.application.DaggerNextcloudTalkApplicationComponent$NextcloudTalkApplicationComponentImpl.viewThemeUtils(DaggerNextcloudTalkApplicationComponent.java:460)
                                                                                                    	at com.nextcloud.talk.application.DaggerNextcloudTalkApplicationComponent$NextcloudTalkApplicationComponentImpl.injectBaseController(DaggerNextcloudTalkApplicationComponent.java:1184)
                                                                                                    	at com.nextcloud.talk.application.DaggerNextcloudTalkApplicationComponent$NextcloudTalkApplicationComponentImpl.inject(DaggerNextcloudTalkApplicationComponent.java:674)
                                                                                                    	at com.nextcloud.talk.controllers.base.BaseController.<init>(BaseController.kt:99)
                                                                                                    	at com.nextcloud.talk.controllers.ConversationsListController.<init>(ConversationsListController.kt:147)
                                                                                                    	at com.nextcloud.talk.controllers.AccountVerificationController.proceedWithLogin$lambda-3(AccountVerificationController.kt:458)
                                                                                                    	at com.nextcloud.talk.controllers.AccountVerificationController.$r8$lambda$Tr4G5lduvSkwbiS824YxO1600xQ(Unknown Source:0)
                                                                                                    	at com.nextcloud.talk.controllers.AccountVerificationController$$ExternalSyntheticLambda1.run(Unknown Source:2)
                                                                                                    	at android.os.Handler.handleCallback(Handler.java:883)
                                                                                                    	at android.os.Handler.dispatchMessage(Handler.java:100)
                                                                                                    	at android.os.Looper.loop(Looper.java:224)
                                                                                                    	at android.app.ActivityThread.main(ActivityThread.java:7590)
                                                                                                    	at java.lang.reflect.Method.invoke(Native Method)
                                                                                                    	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:539)
                                                                                                    	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:950)
2022-12-16 13:11:01.547 20219-20662 WM-WorkerWrapper        com.nextcloud.talk2                  I  Worker result SUCCESS for Work [ id=22d3b1fd-167e-4485-ae35-638e558ac2b6, tags={ com.nextcloud.talk.jobs.SignalingSettingsWorker } ]
2022-12-16 13:11:01.561 20219-20219 Process                 com.nextcloud.talk2                  I  Sending signal. PID: 20219 SIG: 9
```

### Nextcloud Talk app version
fails for 15.0.2 & 15.0.3

works with master (15.1.0 Alpha 12)

**Comment by mahibi** [comment]
it was because theming colors were set to   #fff    instead to   #ffffff

we should add checks/conversions for this case..

**Comment by AndyScherzinger** [comment]
👍

**Comment by AndyScherzinger** [comment]
@mahibi there is a safeguard in android-common 0.3.0, 15.0.x uses v0.1.0, so I would guess that it is just missing the fallback implementation hence it works on 15.1.0

**Comment by mahibi** [comment]
so we can close this issue as "won't fix" as 15.1.0 will be the next release i guess?

**Comment by AndyScherzinger** [comment]
Yes, I would say so

**Comment by AndyScherzinger** [comment]
"fixed in 15.1.0"
